jargons/gitignore.md

```diff
@@ -0,0 +1,32 @@
+---
+title: .gitignore
+
+tags:
+- genel
+- github
```

Comment:
Aslinda bu github'tan ziyade git'in kendisi ile ilgili `github` etiketi de kalabilir ancak `git` etiketini de eklersek bence daha guzel olur.

Reply:
Git etiketi eklendi.